### ISSUE TITLE
README feedback

### ISSUE THREAD

**Issue by vitiral** [vygr/ChrysaLisp#281]
The README is a bit too much of a "laundry list" and I think it could be refocused.

> ChrysaLisp is a 64-bit, MIMD, multi-CPU, multi-threaded, multi-core, multi-user **parallel operating system** with features such as a GUI, terminal, OO Assembler, class libraries, C-Script compiler, Lisp interpreter, debugger, profiler, vector font engine, and more. It supports MacOS, Windows, and Linux for x64, Riscv64 and Arm64 and eventually will move to bare metal.

I've bolded what I think is the key point, but I think another key point is missing here: **self contained**.

## Reword

From what I know of the project, I would reword the opening sentence as (bolded anything I added):

**ChrsyaLisp is a self-contained Parallel Operating System built in a Lisp variant with minimal outside dependencies. It has libraries and applications for:** a GUI, terminal, OO Assembler, class libraries, C-Script compiler, Lisp interpreter, debugger, profiler, vector font engine, **cross-platform IR assembly** and more. It supports MacOS, Windows, and Linux for x64, Riscv64 and Arm64 and eventually will move to bare metal.

## Other applications?

Missing applications, which one of these exist? IMO they are necessary for a "truly bootstrapped" project as they allow you to develop ChrysaLisp in ChrysaLisp without sacrificing your workflow

* Usable text editor
* terminal is mentioned -- is this bash or something else?
* database / structured data storage
* version control and issue management
* web / chat server

As a sidenote, I'm planning on developing all of these as part of civlua. Maybe collaboration can happen :smile: 
* https://github.com/civboot/civlua
* blog post: https://github.com/civboot/civboot/blob/main/blog/0012-dev-libraries-and-software.md

**Comment by vygr** [comment]
Did you try the GUI ?

Editor got extensive work in last release. Doc viewer also worth you taking a look to browse the documentation.

Terminal is based on Lisp command applications in the cmd/ folder. They present a stdin/stdout/stderr style from the Terminal perspective. Take a look at the cmd/ folder for examples.

**Comment by vygr** [comment]
Be happy for you to do a pull request on tidied up README.md ?

As folks will tell you my dyslexia is legendary !

**Comment by paul-hammant** [comment]
@vitiral  If you launch the GUI, there's a terminal opened immediately after login.  And login, for now, is just hit the login button without any typed user/passwd despite the fields being there.

**Comment by vygr** [comment]
The password stuff is nothing but a placeholder ATM !!! Doesn't actually check any passwords file.

The user name does let you have your own environment files with custom settings.

**Comment by vygr** [comment]
Thanks for the feedback and PR.
